pkg/kubectl/cmd/cp_test.go

```diff
@@ -128,6 +128,32 @@ func TestGetPrefix(t *testing.T) {
 	}
 }
 
+func TestStripPathShortcuts(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
+			name:     "test single path shortcut prefix",
+			input:    "../foo/bar",
+			expected: "foo/bar",
+		},
+		{
+			name:     "test multiple path shortcuts",
+			input:    "../../foo/bar",
+			expected: "foo/bar",
+		},
```

Comment:
are the following inputs tested (both as unit tests and that they actually work when used in `cp`):
* absolute paths ("/foo/bar")
* parent paths with no named directory ("../../")
* parent paths with no named directory with no trailing slash ("../..")

Reply:
Thanks, added test in last commit